### PR TITLE
Fix unit tests for `get_target_difficulty`

### PR DESCRIPTION
**Pull request by hansieodendaal**
## Description
- A recent PR increased parameters for `pub fn get_target_difficulty` without changing the unit tests.
- Expanded `inbound_fetch_utxos` unit test to request only a subset plus 1 fake UTXOs.
- Changed log level in `transaction_service/service.rs` to be consistent; not finding a UTXO is not an error in this case.

## Motivation and Context
Unit tests should pass

## How Has This Been Tested?
Ran unit tests locally

## Types of changes

* [X] Bug fix (non-breaking change which fixes an issue)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Feature refactor (No new feature or functional changes, but performance or technical debt improvements)
* [X] New Tests
* [ ] Documentation

## Checklist:

* [X] I'm merging against the `development` branch.
* [X] I ran `cargo-fmt --all` before pushing.
* [X] I ran `cargo test` successfully before submitting my PR.
* [X] I have squashed my commits into a single commit.
* [ ] My change requires a change to the documentation.
* [ ] I have updated the documentation accordingly.
* [ ] I have added tests to cover my changes.

### CODE REVIEW

File: base_layer/core/tests/target_difficulty.rs [hunk 1]
```diff
@@ -196,13 +204,13 @@ fn test_target_block_interval() {
     algos.insert(PowAlgorithm::Blake, PowAlgorithmConstants {
         max_target_time: 240 * 6,
         min_difficulty: 60_000_000.into(),
-        max_difficulty: 60_000_000.into(),
+        max_difficulty: Default::default(),
```
Comment (stringhandler) [inline base_layer/core/tests/target_difficulty.rs]: This will equate to 0, is it correct?

Comment (hansieodendaal) [reply base_layer/core/tests/target_difficulty.rs:207]: The tests passed. I can change it?

Comment (hansieodendaal) [reply base_layer/core/tests/target_difficulty.rs]: Will change to `u64::MAX.into(),`

Comment (hansieodendaal) [reply base_layer/core/tests/target_difficulty.rs]: Done

**Pull request merged**